### PR TITLE
Plugins readme

### PR DESCRIPTION
**Pull request by BibMartin**
A tiny PR to have a doc on plugins, as @andrewgiessel  suggested.

### CODE REVIEW

File: folium/plugins/README.ipynb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    "### Quick and dirty example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is an example of plugin, that imports a Javascript library, defines a css class, creates a new div, defines a Javascript object and add it to the map.\n",
+    "\n",
+    "    from folium.plugins.plugin import Plugin\n",
```
Comment (ocefpaf) [inline folium/plugins/README.ipynb]: Put the code inside a code block like

\`\`\`python
\`\`\`

**Comment by ocefpaf** [comment]
This is awesome, but I wonder if  `folium/plugins/README.ipynb` is the right place (and name) for it.  This is essential "dev docs."

**Comment by BibMartin** [comment]
Please tell me what name/place would be fine.

As for the python block ; I'll try to do that tomorrow.

**Comment by ocefpaf** [comment]
The "right" place does not exist until I do my part and write better docs  :stuck_out_tongue_winking_eye:

**Comment by BibMartin** [comment]
Seems better ?

**Comment by ocefpaf** [comment]
Thanks @BibMartin!  This is very useful!!  Next time we try to sell the plugins we can give a a free tasting first :wink:

**Pull request merged**